src/devtoolsPanel.ts

```diff
@@ -168,6 +169,10 @@ export class DevToolsPanel {
         encodeMessageForChannel((msg) => this.panel.webview.postMessage(msg), "getUrl", { id: request.id, content });
     }
 
+    private async onSocketOpenInEditor(message: string) {
+        // TODO: Parse message and open the requested file

```

Comment:
nit: is there a feature request to link this to?

Reply:
The bug is referenced in the description #55, which is tracking the completion of this function